Classes/Handler/ErrorHandler.php

```diff
@@ -22,153 +36,117 @@ class ErrorHandler
+
+        $options = Hub::getCurrent()->getClient()->getOptions();
+        $options->setEnvironment($this->environment);
+        $options->setRelease($this->getReleaseFromReleaseFile());
+        $options->setProjectRoot(FLOW_PATH_ROOT);
+        $options->setPrefixes([FLOW_PATH_ROOT]);
 
         $this->setTags();
-        $this->setEnvironment();
-        $this->setRelease();
-        $this->setAppPath();
-        $this->setPrefixes();
 
-        $this->emitSentryClientCreated($client);
+        $this->emitSentryClientCreated();
     }
 
     /**
-     * Explicitly handle an exception, should be called from an exception handler (in Flow or TypoScript)
+     * Explicitly handle an exception, should be called from an exception handler (in Flow or Fusion)
      *
      * @param object $exception The exception to capture
      * @param array $extraData Additional data passed to the Sentry sample
      */
-    public function handleException($exception, array $extraData = array())
+    public function handleException($exception, array $extraData = []): void
     {
-        if (!$this->client instanceof \Raven_Client) {
-            return;
-        }
 
         if (!$exception instanceof \Throwable) {
             return;
         }
 
-        $this->setUser();
-
-        $tags = array('code' => $exception->getCode());
-        if ($exception instanceof \Neos\Flow\Exception) {
+        if ($exception instanceof WithReferenceCodeInterface) {
             $extraData['referenceCode'] = $exception->getReferenceCode();
         }
 
-        $this->client->captureException($exception, array(
-                'message' => $exception->getMessage(),
-                'extra' => $extraData,
-                'tags' => $tags
-            )
-        );
+        Hub::getCurrent()->configureScope(function (Scope $scope) use ($exception, $extraData): void {
+            $scope->setUser(['username' => $this->getCurrentUserName()]);
+            $scope->setTag('code', (string)$exception->getCode());
+
+            foreach ($extraData as $extraDataKey => $extraDataValue) {
+                $scope->setExtra($extraDataKey, $extraDataValue);
+            }
+        });
+
+        \Sentry\captureException($exception);
     }
 
     /**
      * Set tags on the raven context
      */
-    protected function setTags()
+    private function setTags(): void
     {
-        $objectManager = Bootstrap::$staticObjectManager;
-        /** @var \Neos\Flow\Utility\Environment $environment */
-        $environment = $objectManager->get('Neos\Flow\Utility\Environment');
-
-        $tags = array(
-            'php_version' => phpversion(),
-            'flow_context' => (string)$environment->getContext(),
-            'flow_version' => FLOW_VERSION_BRANCH
-        );
-
-        $this->client->tags_context($tags);
+        Hub::getCurrent()->configureScope(function (Scope $scope): void {
+            $scope->setTag('php_version', 'phpversion()');
```

Comment:
I guess this should not be enclosed by quotes?

```suggestion
            $scope->setTag('php_version', phpversion());
```

Reply:
Absolutely!!